src/graphql-global-id/src/GlobalID.js

```diff
@@ -27,8 +27,15 @@ export function toGlobalId(type: string, id: string | number): OpaqueIDString {
   return encode(`${type}:${id}`);
 }
 
-// TODO: find out better way how to do it (type should be just an internal detail - see evaluateGlobalIdField)
+/**
+ * TODO: remove in 1.0.0
+ * @deprecated
+ */
 export function __isTypeOf(type: string, opaqueID: mixed): boolean {
+  return isTypeOf(type, opaqueID);
```

Comment:
How about also using `warning` and log here that it is deprecated?

Reply:
Do you think it's necessary? I think we have the usages somehow under control, or not? :) cc @michalsanger

Reply:
necessary, probably not, but I think it is good practise to log this in dev mode. To remind any consumers that it has been deprecated. 

Ok, we don't have that many outside of the 4 of us. So I am also fine with merging it as is.